utils/src/lib.rs

```diff
@@ -1,6 +1,15 @@
 pub mod config;
+pub mod genesis;
 pub mod logging;
 
 // RE-EXPORTS
 // ================================================================================================
 pub use config::Config;
+
+// CONSTANTS
+// ================================================================================================
+
+/// The name of the organization - for config file path purposes
+pub const ORG: &str = "Polygon";
+/// The name of the app - for config file path purposes
+pub const APP: &str = "Miden";
```

Comment:
How do these actually come into play? i.e., are these used only for initializing defaults? In this case, should the default for all files be the same directory as the executable?

Reply:
I continued with Augusto's convention for default file path in the store ([here](https://github.com/0xPolygonMiden/miden-node/blob/e882c4352e69df3f5c5dcf4eff487c253cc097fd/store/src/config.rs#L14-L19)). Putting the `ORG` and `APP` in `utils` was only a manner of necessity: both the `store` and `utils` crate needed it, and store depends on `utils`, so these need to live in `utils`.

The benefit of this design is that the `genesis` utility places the genesis file directly where the store will look for it by default. So the workflow is

```
- run utility
- run `miden-node`
```

instead of 

```
- run utility. The files are now in the local directory
- Remember where the `store` expects to find the genesis file
- Move the JSON file in the expected location
- run `miden-node`
```

What do you think?

Reply:
I think this is fine. One other option could be to move the utility code into `miden-node` crate. May be even under the same binary and we'd be able to do something like:

- `miden-node start` for starting the node.
- `miden-node make-genesis` for creating a genesis file.

But this is more related to https://github.com/0xPolygonMiden/miden-node/pull/107#issuecomment-1857108629

Reply:
Done; moved `GenesisState`, `APP`, `ORG`, etc back in the `miden-node-store` crate